src/Mvc/Mvc.Core/src/RequestSizeLimitAttribute.cs

```diff
@@ -51,4 +52,7 @@ public IFilterMetadata CreateInstance(IServiceProvider serviceProvider)
         filter.Bytes = _bytes;
         return filter;
     }
+
+    /// <inheritdoc />
+    long IRequestSizeLimitMetadata.MaxRequestBodySize => _bytes;
```

Comment:
I forgot `IDisable ` until I saw @Tratcher's [comment](https://github.com/dotnet/aspnetcore/issues/40452#issuecomment-1054591765) on the request size limit middleware issue again. But we should really add an implementation of `DisableRequestSizeLimitAttribute` to `DisableRequestSizeLimitAttribute` too. It would be really weird not to. The implementation would be:

```C#
long? IRequestSizeLimitMetadata.MaxRequestBodySize => null;
```

This aligns with `IHttpMaxRequestBodySizeFeature`. But this means we'll need to update the approved API for `IRequestSizeLimitMetadata` and `DisableRequestSizeLimitAttribute` in #40080. I'll update the issue and mark it read for review Monday.

Reply:
I've implemented `IRequestSizeLimitMetadata` on `DisableRequestSizeLimitAttribute`. I've also added a remark about the security concerns, as per the API review comment.